### ISSUE TITLE
Column names not being picked up

### ISSUE THREAD

**Issue by rcos152** [kometenstaub/csv-to-md#15]
I might be missing this, is there a way to have the formatting for Column headers and then the content in each row have normal text formatting?   The documentation I'm putting together, each row in the csv is the detailed data for the column header.

**Comment by kometenstaub** [comment]
For each column you can choose to apply no formatting, if that's what you're asking.

**Comment by rcos152** [comment]
Not quite...when I apply formatting, it's applied after the column name instead of before so I'm not formatted right when pushing to gh-pages via mkdocs.

**Comment by kometenstaub** [comment]
If you mean inline YAML, it's supposed to be like that.

Can you give an example? I don't exactly understand what outcome you want to achieve.

**Comment by rcos152** [comment]
Sure thing, 

Here's a CSV Example: 
Recommendation Number,SP Rating,Title,Description,Remediation Procedure,Comcast Site Policy,Is there a change,What's changed
0.1,3,Ensure RunID level is set to multiple user multiple user mode. Check the operating state of the LINUX or UNIX-based and confirm that the workstation is running in multiple user mode under the command line interface and not under the graphical user interface,,,Yes,New control

OUtput of .md file attached.
[0.1,New co.md](https://github.com/kometenstaub/csv-to-md/files/8661399/0.1.New.co.md)
